documentation/src/main/asciidoc/migration/index.asciidoc

```diff
@@ -118,3 +118,187 @@ used to match no documents with the Lucene backend, but all documents with the E
 A boolean predicate with no clause will now consistently match no documents regardless of the backend.
 * API methods matching `*Async(..)` pattern (e.g. `SearchWorkspace#purgeAsync()`) will no longer block
 if internal queues of operations are full, but will throw `RejectedOperationException` instead.
+
+[[outboxpolling]]
+== Outbox polling system tables
+
+Due to changes to the entity that represents the outbox event, DB schema changes are required.
+A new column -  `created::timestamp` is introduced, `id` column changes the type from integer to varchar.
+You can find suggested migration scripts for the supported databases below:
+
+.Postgresql:
+[,sql]
+----
+-- add `created` column that will be used for sorting of events:
+alter table hsearch_outbox_event
+    add column created timestamp not null default now();
```

Comment:
You're setting a default value here; is that part of the schema? Because it shouldn't be, as the schema generated by Hibernate Search doesn't include it.

Maybe you shouldn't use `not null`/`default` here, and just add an `alter` statement later to add the `not null` constraint?